README.md

```diff
@@ -51,40 +44,30 @@ module "example" {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
-| admin_pw | The password for the Kerberos admin role. | string | | yes |
 | aws_region | The AWS region to deploy into (e.g. us-east-1). | string | | yes |
 | cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | string | `cool-certificates` | no |
-| cert_role_arn | The ARN of the role to assume when creating a role to allow reading certboto certificate data (e.g. "arn:aws:iam::123456789012:role/ReadCert-example.com"). | string | | yes |
+| cert_create_read_role_arn | The ARN of the role to assume when creating a role to allow reading certboto certificate data (e.g. "arn:aws:iam::123456789012:role/CertCreateReadRole"). | string | | yes |
+| client_network | A string containing the network and netmask to assign client addresses. The server will take the first address. (e.g. "10.240.0.0 255.255.255.0"). | string | | yes |
```

Comment:
This input is marked as required but it is not in the example usage above.

Reply:
Resolved in commit 48850c3.